charts/kusion/README.md.gotmpl

```diff
@@ -57,16 +63,16 @@ helm install kusion-release ./kusion
 First, you need to base64 encode your kubeconfig files.
```

Comment:
The kubeconfig key naming convention changed from 'kubeconfig0' to 'kubeconfig-0'. This is a breaking change that should be clearly documented in the release notes as it could affect existing deployments.